.circleci/config.yml

```diff
@@ -30,10 +38,34 @@ jobs:
       - checkout
       - node/with-cache:
           steps:
-            - run: ./scripts/init-env.sh STAGING_
+            - run:
+                command: |
+                  if [ $CIRCLE_BRANCH = "staging" ]; then
```

Comment:
if the environment variables are preserved between commands, you should set PREFIX, ENV, and TARGET in one conditional

Reply:
I don't think they are.

Reply:
why not making it a single command then

Reply:
Confirmed (https://circleci.com/gh/sparkletown/sparkle/938?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link).

The environment setup is the same for the functions and hosting, that's why it's separated from the the actual deploy part. I can group them.